iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TopologyLevel.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2017, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Change copyright information to "Copyright (c) 2017, RTE (http://www.rte-france.com)"